src/Console/ServeCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace TightenCo\Jigsaw\Console;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ServeCommand extends Command
+{
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('serve')
+            ->setDescription('Serve local site with php built-in server.')
+            ->addArgument(
```

Comment:
Nah let's leave the hostname option out for now, if it ends up being a useful addition we can tackle it in another PR. Let's keep this PR just about the serve command, and make the switch to the build command in another PR too.